chia/types/spend_bundle.py

```diff
@@ -21,9 +21,13 @@ class SpendBundle(Streamable):
     between transactions are more flexible than in bitcoin).
     """
 
-    coin_spends: List[CoinSpend]
+    coin_solutions: List[CoinSpend]
```

Comment:
I didn't notice this until now. I see what's happening now. I'd actually recommend against this, since we actually *want* `coin_spends` to be the "official" way, so the "hack" code should all be focussed around making the deprecated way working.